src/fft.js

```diff
@@ -99,7 +110,6 @@ define(function (require) {
     // default connections to p5sound fftMeter
     p5sound.fftMeter.connect(this.analyser);
 
-    this.analyser.smoothingTimeConstant = this.smoothing;
```

Comment:
Wouldn't it do the same thing? In the arrangement in the PR wouldn't it define the getters and setters, and then, in essence, set `this.analyser.smoothingTimeConstance = smoothing || 0.8`?